dataverse-webapp/src/main/webapp/dataset.xhtml

```diff
@@ -1362,6 +1362,7 @@
                                 </label>
                                 <div class="col-sm-8">
                                     <p:inputText id="replyTo"
+                                                 styleClass="stretched"
```

Comment:
`styleClass="form-control"` will have the same effect and it is alredy in use in many places